lib/Polaris.Class.ps1

```diff
@@ -1,383 +1,394 @@
-            $this.Listener.TimeoutManager.RequestQueue = [timespan]::FromMinutes(5)
-            $this.Listener.TimeoutManager.IdleConnection = [timespan]::FromSeconds(45)
-            $this.Listener.TimeoutManager.EntityBody = [timespan]::FromSeconds(50)
-            $this.Listener.TimeoutManager.HeaderWait = [timespan]::FromSeconds(5)
-        }
-
-        $this.Listener.Start()
-    }
-
-    static [void] Send (
-        [System.Net.HttpListenerResponse]$RawResponse, 
-        [PolarisResponse]$Response
-    ) {
-        if ($Response.StreamResponse) {
-            [Polaris]::Send(
-                $RawResponse,
-                $Response.StreamResponse,
-                $Response.StatusCode,
-                $Response.ContentType,
-                $Response.Headers
-            )
-        }
-        else {
-            [Polaris]::Send(
-                $RawResponse,
-                $Response.ByteResponse,
-                $Response.StatusCode,
-                $Response.ContentType,
-                $Response.Headers
-            )
-        }
-    }
-
-    static [void] Send (
-        [System.Net.HttpListenerResponse]$RawResponse, 
-        [byte[]]$ByteResponse, 
-        [int]$StatusCode, 
-        [string]$ContentType, 
-        [System.Net.WebHeaderCollection]$Headers
-    ) {
-        $RawResponse.StatusCode = $StatusCode;
-        $RawResponse.Headers = $Headers;
-        $RawResponse.ContentType = $ContentType;
-        $RawResponse.ContentLength64 = $ByteResponse.Length;
-        $RawResponse.OutputStream.Write($ByteResponse, 0, $ByteResponse.Length);
-        $RawResponse.OutputStream.Close();
-    }
-    
-    static [void] Send (
-        [System.Net.HttpListenerResponse]$RawResponse, 
-        [System.IO.Stream]$StreamResponse, 
-        [int]$StatusCode, 
-        [string]$ContentType, 
-        [System.Net.WebHeaderCollection]$Headers
-    ) {
-        $RawResponse.StatusCode = $StatusCode;
-        $RawResponse.Headers = $Headers;
-        $RawResponse.ContentType = $ContentType;
-        $StreamResponse.CopyTo($RawResponse.OutputStream);
-        $RawResponse.OutputStream.Close();
-    }
-
-    static [void] Send (
-        [System.Net.HttpListenerResponse]$RawResponse,
-        [byte[]]$ByteResponse,
-        [int]$StatusCode,
-        [string]$ContentType
-    ) {
-        [Polaris]::Send($RawResponse, $ByteResponse, $StatusCode, $ContentType, $Null)
-    }
-
-    [void] Log ([string]$LogString) {
-        try {
-            $this.Logger($LogString)
-        }
-        catch {
-            Write-Host $_.Message
-            Write-Host $LogString
-        }
-    }
-
-
-    Polaris (
-        [Action[string]]$Logger
-    ) {
-        if ($Logger) {
-            $this.Logger = $Logger
-        }
-        else {
-            $this.Logger = {
-                param($LogItem)
-                Write-Host $LogItem
-            }
-        }
-
-    }
-}
+class Polaris {

```

Comment:
This diff looks off. It's showing you modified the entire file instead of highlighting just the changes that you made. If you're using VSCode could you make sure it didn't flip the file from LF endings to CRLF? It should look like this:

![image](https://user-images.githubusercontent.com/6989492/46349281-9a4fb180-c61f-11e8-86f9-a60b93855979.png)